apps/volunteer/shifts/bar.py

```diff
@@ -0,0 +1,98 @@
+from ..event_date import edt
+
+bar_shifts = {
+    "Bar": {
+        "Bar": [
+            {
+                "first": edt("thur", "15:00:00"),
+                "final": edt("fri", "01:00:00"),
+                "min": 1,
+                "max": 3,
+                "changeover": 10,
+            },
+            {
+                "first": edt("thur", "16:00:00"),
+                "final": edt("fri", "00:00:00"),
+                "min": 1,
+                "max": 3,
+                "changeover": 10,
+            },
+            {
+                "first": edt("fri", "11:00:00"),
+                "final": edt("sat", "01:00:00"),
+                "min": 1,
+                "max": 3,
+                "changeover": 10,
+            },
+            {
+                "first": edt("fri", "12:00:00"),
+                "final": edt("sat", "00:00:00"),
+                "min": 1,
+                "max": 3,
+                "changeover": 10,
+            },
+            {
+                "first": edt("sat", "11:00:00"),
+                "final": edt("sun", "01:00:00"),
+                "min": 1,
+                "max": 3,
+                "changeover": 10,
+            },
+            {
+                "first": edt("sat", "12:00:00"),
+                "final": edt("sun", "00:00:00"),
+                "min": 1,
+                "max": 3,
+                "changeover": 10,
+            },
+            {
+                "first": edt("sun", "11:00:00"),
+                "final": edt("mon", "01:00:00"),
+                "min": 1,
+                "max": 3,
+                "changeover": 10,
+            },
+            {
+                "first": edt("sun", "12:00:00"),
+                "final": edt("mon", "00:00:00"),
+                "min": 1,
+                "max": 3,
+                "changeover": 10,
+            },
+        ]
+    },
+    "Cybar": {
+        "Cybar": [
```

Comment:
I've expanded times to 18:00-2:00 for Fri, 10:00-2:00 for Sat and Sun.

I've haven't put in split/overlapping shifts since that doesn't really make sense with 1-2 people (can't swap out half a person...). But should we actually have more people than that here since it is a bigger bar than last time?